redisinsight/api/config/default.ts

```diff
@@ -92,18 +97,25 @@ export default {
   plugins: {
     stateMaxSize: parseInt(process.env.PLUGIN_STATE_MAX_SIZE, 10) || 1024 * 1024,
   },
-  enablementArea: {
-    updateUrl: process.env.ENABLEMENT_AREA_UPDATE_URL
+  guides: {
+    updateUrl: process.env.GUIDES_UPDATE_URL
       || 'https://github.com/RedisInsight/Guides/releases/download/latest',
-    zip: process.env.ENABLEMENT_AREA_ZIP || 'data.zip',
-    buildInfo: process.env.ENABLEMENT_AREA_CHECKSUM || 'build.json',
+    zip: process.env.GUIDES_ZIP || 'data.zip',
+    buildInfo: process.env.GUIDES_CHECKSUM || buildInfoFileName,
     devMode: !!process.env.GUIDES_DEV_PATH,
   },
+  tutorials: {
+    updateUrl: process.env.TUTORIALS_UPDATE_URL
+      || 'https://github.com/RedisInsight/Tutorials/releases/download/latest',
+    zip: process.env.TUTORIALS_ZIP || 'data.zip',
+    buildInfo: process.env.TUTORIALS_CHECKSUM || buildInfoFileName,
+    devMode: !!process.env.TUTORIALS_DEV_PATH,
+  },
   content: {
     updateUrl: process.env.CONTENT_UPDATE_URL
       || 'https://github.com/RedisInsight/Statics/releases/download/latest',
     zip: process.env.CONTENT_ZIP || 'data.zip',
-    buildInfo: process.env.CONTENT_CHECKSUM || 'build.json',
+    buildInfo: process.env.CONTENT_CHECKSUM || buildInfoFileName,
```

Comment:
If you decided to move `build.json` to a constant why you ignored `data.zip`?

Reply:
Just linter highlight. And i move it to constant string. But yea, i'll update 'data.zip' too